DailyReportWeb/Controllers/Api/ConfirmDraftController.cs

```diff
@@ -58,7 +61,7 @@ private string GetDraftConfirmationDetails(ExecutionContext context, AdvancedRep
                 var basicSettingsId = db.BasicSettings.Single(bs => bs.UniqueProjectKey == context.Id).Id;
                 var reportSentDate = db.ReportExecutionSummaries.Single(res => res.BasicSettingsId == basicSettingsId).LastFinalReportSentDate;
 
-                if (reportSentDate != null && reportSentDate.Value.Date == DateTime.Today)
+                if (reportSentDate != null && reportSentDate.Value.ToOriginalTimeZone(offsetFromUtc).Date == DateTime.Now.ToOriginalTimeZone(offsetFromUtc).Date)
```

Comment:
Eu am folosit aia sa evit multiple verificari, inclusiv cea pentru null,
plus ca e un date, ar trebui sa folosesti metodele specifice, zic...

On Tue, May 19, 2015 at 12:36 PM, Sebastian Dumitrascu <
notifications@github.com> wrote:

> In DailyReportWeb/Controllers/Api/ConfirmDraftController.cs
> <https://github.com/equilobe/DailyReportingTools/pull/29#discussion_r30583232>
> :
>
> > @@ -58,7 +61,7 @@ private string GetDraftConfirmationDetails(ExecutionContext context, AdvancedRep
> >                  var basicSettingsId = db.BasicSettings.Single(bs => bs.UniqueProjectKey == context.Id).Id;
> >                  var reportSentDate = db.ReportExecutionSummaries.Single(res => res.BasicSettingsId == basicSettingsId).LastFinalReportSentDate;
> >
> > -                if (reportSentDate != null && reportSentDate.Value.Date == DateTime.Today)
> > +                if (reportSentDate != null && reportSentDate.Value.ToOriginalTimeZone(offsetFromUtc).Date == DateTime.Now.ToOriginalTimeZone(offsetFromUtc).Date)
>
> Pai eu am folosit doar == . Vad ca in metodele noi, scrise de Vlad, a
> facut verificarile cu DateTime.Compare. O sa modific sa fie aceeasi metoda
> peste tot.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/equilobe/DailyReportingTools/pull/29/files#r30583232>.
>



-- 
*Vlad Ichimescu*
*IT Specialist* | www.vladichimescu.me | Mobile: +40 769 584 584
<+40769584584>
*Software Developer* | Equilobe Software | www.equilobe.com | 75 Tunari
Street, 1st floor